mlflow/cli.py

```diff
@@ -234,8 +234,10 @@ def _validate_static_prefix(ctx, param, value):  # pylint: disable=unused-argume
               help="Additional command line options forwarded to gunicorn processes.")
 @click.option("--waitress-opts", default=None,
               help="Additional command line options for waitress-serve.")
+@click.option("--activate-prometheus", default=False, is_flag=True,
+              help="Activate prometheus exporter to expose metrics on /metrics endpoint.")
```

Comment:
Might be better to call this "enable-prometheus" or "expose-prometheus" since it's a more common way to name flags. We don't really run Prometheus ourselves.

Reply:
I agree, it is clearer with your recommandation.